utils/enums.py

```diff
@@ -1,27 +1,30 @@
 from enum import Enum
 
+
 class UserStatus(str, Enum):
-    
     TO_CONTACT = "TO CONTACT"
     IN_REVIEW = "IN REVIEW"
+    TASK_SENT = "TASK SENT"
```

Comment:
I was thinking the Task sent is same as contacted and task submitted is same as in review